DESCRIPTION

```diff
@@ -13,12 +13,6 @@ Authors@R: c(
   person("Aron", "Atkins", role = "aut", email = "aron@rstudio.com"),
   person("Rob", "Hyndman", role = "aut", email = "robjhyndman@gmail.com"),
   person(family = "RStudio, Inc.", role = "cph"),
-  person(family = "jQuery Foundation", role = "cph",
```

Comment:
We are still using jquery just don't need the explicit paring with your JS file (i.e. it's already included elsewhere automatically). The reason we had it there for stickytableheaders was to ensure it was included in the correct order (i.e. before stickytableheaders).